# disturbance estimate, 3 components
float64[3] d_est
float64 time